pages.fr/common/docker-load.md

```diff
@@ -0,0 +1,16 @@
+# docker load
+
+> Charger des images docker depuis des fichiers ou `stdin`.
+> Plus d'informations: <https://docs.docker.com/engine/reference/commandline/load/>.
```

Comment:
```suggestion
> Plus d'informations: <https://docs.docker.com/engine/reference/commandline/load>.
```

Reply:
This should probably be removed from the English page too, or else the test will fail. Will do it now.